soccer/gameplay/skills/goalside_mark.py

```diff
@@ -0,0 +1,163 @@
+import single_robot_behavior
+import behavior
+import robocup
+import main
+import constants
+import role_assignment
+
+
+class Goalside_Mark(single_robot_behavior.SingleRobotBehavior):
+    '''
+    The goalside mark will take a robot or mark point and position itself on line with the best shot from 
+    the position of the robot/mark_point.
+    @param: ratio - this will control how far from the opposing robot the defender will mark (0 is close to robot)
+    '''
+
+    def __init__(self):
+        super().__init__(continuous=True)
+        #Below params are described above @properties
+        self._ratio = 0.2
+        #This threshold is lower than mark to ensure we get goal side of the opponent fast, 
+        #then back off based on ratio
+        
+        self._mark_line_thresh = constants.Robot.Radius*4
+        self._mark_robot = None 
+        self._mark_point = None 
+
+        self._target_point = None 
+
+        self.add_transition(behavior.Behavior.State.start,
+                            behavior.Behavior.State.running, lambda: True,
+                            "immediately")
+        self.kick_eval = robocup.KickEvaluator(main.system_state())
+        self.mark_pos = None
+        self.adjusted_mark_pos = None
+
+    def execute_running(self):
+        #pylint: disable=no-member
```

Comment:
Which variable is giving the pylint problem?

Reply:
this is the one thing I'm not sure how to deal with- don't know how pylint works or what its looking for

Reply:
So I removed this and ran `make pylint`. There's a bunch of crap that comes up, but nothing similar to the `self.mark_robot` has no member `pos` that came up earlier. Try taking it out and seeing if circle ci blows up.

Reply:
when I run make pylint - the only thing that comes up is related to a network util file that I've definitely never touched